### DISCUSSION TITLE
Reorder columns in tables

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#1397] [Ideas]
### Describe the feature or potential improvement

When adding more and more columns to tables, it'd be useful to be able to reorder them

**Comment by clemra** [comment]
Reorder and resize -- let's go!

**Comment by marliessophie** [comment]
Columns can be reordered as of [Lanfguse version 2.79.0](https://github.com/langfuse/langfuse/releases/tag/v2.79.0). Marking this as done, while keen to hear feedback on the functionality!
